File: jadx-plugins/jadx-xapk-input/src/main/java/jadx/plugins/input/xapk/data/XApkManifest.java
package jadx.plugins.input.xapk.data;

import java.util.List;

import com.google.gson.annotations.SerializedName;

public class XApkManifest {
	@SerializedName("xapk_version")
	int version;
	@SerializedName("split_apks")
	List<SplitApk> splitApks;

	public List<SplitApk> getSplitApks() {
		return splitApks;
	}

	public void setSplitApks(List<SplitApk> splitApks) {
		this.splitApks = splitApks;
	}

	public int getVersion() {
		return version;
	}

	public void setVersion(int version) {
		this.version = version;
	}
}

File: jadx-plugins/jadx-xapk-input/src/main/java/jadx/plugins/input/xapk/XapkManifest.kt
package jadx.plugins.input.xapk

import com.google.gson.annotations.SerializedName

data class XapkManifest(
	@SerializedName("xapk_version")
	var xapkVersion: Int = 0,
	@SerializedName("split_apks")
	var splitApks: List<SplitApk> = listOf(),
) {
	data class SplitApk(
		@SerializedName("file")
		var file: String = "",
		@SerializedName("id")
		var id: String = "",
	)
}
